Library/Homebrew/livecheck_strategy/apache.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module LivecheckStrategy
+  class Apache
```

Comment:
```suggestion
  module Homebrew::Livecheck::Strategies::Apache
```

and put it in `livecheck/strategies/apache` using `module_function` to avoid `self.`

Reply:
We have `Homebrew::Livecheck::Strategy` above and `Homebrew::Livecheck::Strategies::Apache` here. Just to be sure, is the `Strategy` vs. `Strategies` mismatch intentional or should we be doing `Homebrew::Livecheck::Strategy::Apache` here?

Reply:
`Strategy` probably makes more sense to me.

Reply:
I agree. It makes sense for `LivecheckStrategy` to become `Homebrew::Livecheck::Strategy` and for `LivecheckStrategy::Apache` to become `Homebrew::Livecheck::Strategy::Apache`.

Last question on this: It seems like we generally align the name of the Ruby file and the related directory (e.g., `version.rb`, `version`). Should we also use a `livecheck/strategy/` directory name, to align with `livecheck/strategy.rb` (as discussed in the other comment)?

Reply:
> Should we also use a `livecheck/strategy/` directory name

Yup 👍🏻